package.json

```diff
@@ -73,6 +73,7 @@
     "@commitlint/cli": "^6.1.3",
     "@commitlint/config-angular": "^6.1.3",
     "@commitlint/travis-cli": "^6.1.3",
+    "@cypress/webpack-preprocessor": "^2.0.1",
```

Comment:
This is needed for webpack 4.x support - otherwise the error `Cannot read property 'afterCompile' of undefined` would be thrown when starting cypress.